modules/tinymce/src/themes/silver/test/ts/browser/editor/core/UndoRedoTest.ts

```diff
@@ -0,0 +1,85 @@
+import { ApproxStructure, Assertions, Keys, UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarBody } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.core.UndoRedoTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: 'redo undo',
+    menubar: true
+  }, []);
+
+  const assertToolbarButtonDisabled = (title: string, state: boolean = true) =>
+    UiFinder.exists(SugarBody.body(), `button[title="${title}"][aria-disabled="${state}"]`);
+
+  const pAssertMenuItemDisabled = (editor, item: string, state: boolean = true) =>
+    TinyUiActions.pWaitForUi(editor, `div[title="${item}"][role="menuitem"][aria-disabled="${state}"]`);
+
+  const pAssertUiDisabled = async (editor: Editor, disabled: boolean) => {
```

Comment:
None of these are checking if it's solely disabled now... can you please rename these to reflect they are checking the state instead and then remove the default values for the state to make this clearer to read in the tests?

Reply:
I changed their name. Hope it is easier to read now

Reply:
Looks good, thanks 👍